.github/workflows/build-test.yaml

```diff
@@ -1290,5 +1317,7 @@ jobs:
       - validate-yamlescape
       - validate-no-redeploy-on-restart
       - validate-kubernetes-installer-preflight
+      # cli-only tests
```

Comment:
wanna move the `validate-kots-pull` test to this section as well?

Reply:
kots pull does deploy stuff... this one is a one off right now because it does not run on a k3s matrix.  there is no kubernetes at all.

Reply:
ah ok. makes sense.